code/game/objects/items/clown_items.dm

```diff
@@ -246,3 +246,47 @@
 	desc = "Just looking at this makes you want to giggle."
 	icon_state = "laughter"
 	list_reagents = list(/datum/reagent/consumable/laughter = 50)
+
+/obj/item/rubber_chicken
+	name = "rubber chicken"
+	desc = " A robust rubber chicken the size of a Louisiana broiler.\nIt looks somewhat clammy and exudes a noticable rubbery smell.\nA small peeling sticker on the chicken's left thigh reads: 'made by waffle co.'"
```

Comment:
shouldn't "made by waffle co." be capitalized?

Reply:
done